eq-author/src/App/page/Design/Validation/AnswerValidation.js

```diff
@@ -57,7 +57,13 @@ const formatValue = (value, { type, properties }) => {
     return `£${value}`;
   }
   if (type === UNIT) {
-    return `${value} ${unitConversion[properties.unit].abbreviation}`;
+    let valueWithUnit;
+    unitConversion[properties.unit] == null
+      ? (valueWithUnit = `${value}`)
+      : (valueWithUnit = `${value} ${
+          unitConversion[properties.unit].abbreviation
+        }`);
+    return valueWithUnit;
```

Comment:
```suggestion
    return unitConversion[properties.unit] == null
      ? `${value}`
      :  `${value} ${
          unitConversion[properties.unit].abbreviation
        }`;
    
```

Does that work instead?